fabfile/setup.py

```diff
@@ -168,7 +217,8 @@ def redhat_install_packages():
 
     with hide('stdout'):
         sudo('yum install -q -y libxml2-devel libxslt-devel'
-            ' openssl-devel pam-devel readline-devel libcurl-devel git libzstd-devel lz4-devel')
+            ' openssl-devel pam-devel readline-devel libcurl-devel'
+            ' git libzstd-devel lz4-devel perl-IPC-Run perl-Test-Simple')
```

Comment:
I have added them to support extensions which has test suites which is run by perl's tap framework. We currently support extensions whose tests can be run by pg_regress. But some extensions have tests that can be run by perl(Postgres calls tap tests). 

We currently cannot run perl tests properly because I could not find a proper way to load extensions during testing the extension. (there is no flag like --load-extension for tap tests)